qp_qiime2/tests/test_qiime2.py

```diff
@@ -76,6 +79,114 @@ def test_not_analysis_artifact(self):
 
+    def test_feature_classifier(self):
+        dbpath = join(self.basedir, '..', '..', 'databases',
+                      'gg-13-8-99-515-806-nb-classifier.qza')
+        original_params = {
+            'qp-hide-method': 'classify_sklearn',
+            'qp-hide-plugin': 'feature-classifier',
+            'The feature data to be classified.': '8',
+            'qp-hide-paramDirection of reads with respect to reference '
+            'sequences. same will cause reads to be classified unchanged; '
+            'reverse-complement will cause reads to be reversed and '
+            'complemented prior to classification. Default is to autodetect '
+            'based on the confidence estimates for the first 100 reads. '
+            '(read_orientation)': 'read_orientation',
+            'Direction of reads with respect to reference sequences. same '
+            'will cause reads to be classified unchanged; reverse-complement '
+            'will cause reads to be reversed and complemented prior to '
+            'classification. Default is to autodetect based on the confidence '
+            'estimates for the first 100 reads. (read_orientation)':
+            'reverse-complement', 'qp-hide-paramConfidence threshold for '
+            'limiting taxonomic depth. Provide -1 to disable confidence '
+            'calculation, or 0 to calculate confidence but not apply it to '
+            'limit the taxonomic depth of the assignments. (confidence)':
+            'confidence', 'Confidence threshold for limiting taxonomic depth. '
+            'Provide -1 to disable confidence calculation, or 0 to calculate '
+            'confidence but not apply it to limit the taxonomic depth of the '
+            'assignments. (confidence)': '0.7', 'qp-hide-param"all" or '
+            'expression, as in "3*n_jobs". The number of batches (of tasks) '
+            'to be pre-dispatched. (pre_dispatch)': 'pre_dispatch', '"all" or '
+            'expression, as in "3*n_jobs". The number of batches (of tasks) '
+            'to be pre-dispatched. (pre_dispatch)': '2*n_jobs',
+            'qp-hide-paramThe maximum number of concurrently worker '
+            'processes. If -1 all CPUs are used. If 1 is given, no parallel '
+            'computing code is used at all, which is useful for debugging. '
+            'For n_jobs below -1, (n_cpus + 1 + n_jobs) are used. Thus for '
+            'n_jobs = -2, all CPUs but one are used. (n_jobs)': 'n_jobs',
+            'The maximum number of concurrently worker processes. If -1 all '
+            'CPUs are used. If 1 is given, no parallel computing code is '
+            'used at all, which is useful for debugging. For n_jobs below -1, '
+            '(n_cpus + 1 + n_jobs) are used. Thus for n_jobs = -2, all CPUs '
+            'but one are used. (n_jobs)': '1', 'qp-hide-paramNumber of reads '
+            'to process in each batch. If 0, this parameter is autoscaled to '
```

Comment:
autoscaled -> auto-scaled?

Reply:
This comes from Q2, if you feel strongly about it, please open an issue directly in the plugin feature-classifier.